Sources/ArcGISToolkit/Resources/ar.lproj/Localizable.strings

```diff
@@ -644,6 +715,10 @@ The first and second parameter hold the minimum and maximum values respectively.
 /* E.g. Open a hyperlink. */
 "View" = "عرض";
 
+/* A label indicating a utility association's 
+containment is visible. */
+"Visible Content" = "بيت_Visible Content________________لاحقة";
```

Comment:
Translation for this pseudo string expected before July 29.

Reply:
Merged in [36db45b](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/pull/1221/commits/36db45b42fd5514c5f54a03a729477a5c42b8be4)